programmers/난이도별/level01.같은_숫자는_싫어/sjjyy.java

```diff
@@ -0,0 +1,33 @@
+package 난이도별.level01.같은_숫자는_싫어;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class sjjyy
+{
+    public static int [] solution(int[] array)
+    {
+        ArrayList<Integer> arrayList = new ArrayList<>();
+        int len = array.length - 1;
+
+        for(int i = 0 ; i < len ; i++)
+        {
+            if(array[i] != array[i+1])
+                arrayList.add(array[i]);
+        }
+
+        arrayList.add(array[len]); // 마지막 인덱스 값
+
+        int aLen = arrayList.size();
+        int[] ans = new int[aLen];
+        for(int i = 0 ; i < aLen ; i++) ans[i] = arrayList.get(i);
+
+        return ans;
```

Comment:
```suggestion
        return arrayList.toArray();
```
List는 이렇게 Array로 변경할 수 있습니다!

Reply:
함수 반환형이 int[]라 이렇게 하면 형식이 안 맞지 않나요...?! 🤔

Reply:
@sjjyy 음 List<Integer>는 int[]로 변환 가능합니다!

Reply:
아 조금 더 찾아보니 Integer에서 int로 바로 변환은 안되고 다름과 같은 과정이 필요하네요!
```java
    List<Integer> list = Arrays.asList(1,2,3,4);
    int[] arr = list.stream().mapToInt(Integer::valueOf)
                             .toArray();
```

혼란을 드려서 죄송합니다!